repositories/ingested_data_read_repository.go

```diff
@@ -305,9 +311,11 @@ func createQueryAggregated(exec Executor, tableName string,
 		// COUNT(*) is a special case, as it does not take a field name (we do not want to count only non-null
 		// values of a field, but all rows in the table that match the filters)
 		selectExpression = "COUNT(*)"
-	} else {
+	} else if fieldType == models.Int {
```

Comment:
this and below is the real change, everything else is just injecting the parameter